timdex_dataset_api/dataset.py

```diff
@@ -93,21 +93,11 @@ def row_count(self) -> int:
 
     def load(
         self,
-        *,
-        timdex_record_id: str | None = None,
-        source: str | None = None,
-        run_date: str | date | None = None,
-        run_type: str | None = None,
-        run_id: str | None = None,
-        action: str | None = None,
-        year: str | None = None,
-        month: str | None = None,
-        day: str | None = None,
+        **filters: Unpack[DatasetFilters],
```

Comment:
This use of `Unpack` was new to me.  If interested, [reading / skimming PEP 692 was really quite interesting](https://peps.python.org/pep-0692/).

Reply:
New to me as well, very clean and efficient!